tutorials/cifar10-distributed.ipynb

```diff
@@ -0,0 +1,1319 @@
+      },
+      "source": [
+        "Specify `with_clearml=True` in `config` below and monitor the experiment on the dashboard. "
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "09xlncFZfmeS"
+      },
+      "source": [
+        "## Download Data\n",
+        "\n",
+        "Let's download our data first which can later be used by all the processes to instantiate our dataloaders. The following command will download the CIFAR10 dataset to a folder `cifar10`."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "gRpACWtUfyn1"
+      },
+      "source": [
+        "!python -c \"from torchvision.datasets import CIFAR10; CIFAR10('cifar10', download=True)\""
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "D71VkD74he9J"
+      },
+      "source": [
+        "## Common Configuration\n",
+        "\n",
+        "We maintain a `config` dictionary which can be extended or changed to store parameters required during training. We can refer back to this code when we will use these parameters later."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "9bg7unvqhegL"
+      },
+      "source": [
+        "config = {\n",
+        "    \"seed\": 543,\n",
+        "    \"data_path\": \"cifar10\",\n",
+        "    \"output_path\": \"output-cifar10/\",\n",
+        "    \"model\": \"resnet18\",\n",
+        "    \"batch_size\": 512,\n",
+        "    \"momentum\": 0.9,\n",
+        "    \"weight_decay\": 1e-4,\n",
+        "    \"num_workers\": 2,\n",
+        "    \"num_epochs\": 5,\n",
+        "    \"learning_rate\": 0.4,\n",
+        "    \"num_warmup_epochs\": 1,\n",
+        "    \"validate_every\": 3,\n",
+        "    \"checkpoint_every\": 200,\n",
+        "    \"backend\": None,\n",
+        "    \"resume_from\": None,\n",
+        "    \"log_every_iters\": 15,\n",
+        "    \"nproc_per_node\": None,\n",
+        "    \"with_clearml\": False,\n",
+        "    \"with_amp\": False,\n",
+        "}"
+      ],
+      "execution_count": 8,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "DzuG8QAr5Djf"
+      },
+      "source": [
+        "## Basic Setup"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "bIgzky7Q7kUk"
+      },
+      "source": [
+        "### Imports"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "0RVISbXd_h1F"
+      },
+      "source": [
+        "import os\n",
+        "from datetime import datetime\n",
+        "from pathlib import Path\n",
+        "\n",
+        "import torch\n",
+        "import torch.nn as nn\n",
+        "import torch.optim as optim\n",
+        "from torchvision import datasets, models\n",
+        "from torchvision.transforms import (\n",
```

Comment:
Why not `from torchvision import datasets, models, transforms` ? Anyway, that's a detail.

Reply:
I find it easier to use `Compose` rather than `transforms.Compose` etc later in the code. Should I still this?

Reply:
Yes, let's keep it as `Compose`